<launch>
  <arg name="enable_motor" default="true"/>
  <arg name="method"         default="0"/>

  <node name="PavoPcdNode"     pkg="pavo2_ros"  type="pavo2_pcd_node"  output="screen">
    <param name="frame_id"     type="string"   value="pcd_frame"/><!--设置Pointcloud的id,default=pcd_frame-->
    <param name="cloud_topic"  type="string"   value="cloud"/><!--设置Pointcloud的topic,default=cloud-->
    <param name="angle_min"    type="double"   value="-140.00" /><!--设置最小角度，单位度.取值范围[-140,140].default=-140.0-->
    <param name="angle_max"    type="double"   value="140.00"/><!--设置最大角度，单位度.取值范围[-140,140].default=140.0-->
    <param name="range_min"    type="double"   value="0.10" /><!--设置最小距离，单位米，default=0.10-->
    <param name="range_max"    type="double"   value="50.0" /><!--设置最大距离，单位米，default=20-->
    <param name="angle_resolution"  type="int"      value="32"/>
    <param name="motor_speed"  type="int"      value="50" /><!--default=50Hz-->
    <param name="angle_start"  type="int"      value="4000" /><!--设置起始角度，default=4000，即40度-->
    <param name="angle_end"  type="int"      value="32000" /><!--设置结束角度，default=32000，即320度-->
    <param name="intensity"     type="bool"     value="true"/><!--强度使能，使能强度true，不使能强度false，default=true -->
    <param name="inverted"     type="bool"     value="false"/><!--配置雷达是否倒装，倒装true，正装false-->
    <param name="echo_mode"     type="bool"     value="true"/><!--二回波使能，使能二回波true，不使能二回波false，default=true -->
    
    <param name="lidar_ip"     type="string"   value="10.10.10.121" /><!--所要连接的雷达IP地址，即通过PavoView设置的雷达IP地址-->
    <param name="lidar_port"   type="int"      value="2368" /><!--雷达ip默认为10.10.10.121，端口号默认为2368-->
  </node>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find pavo2_ros)/rviz/pavo2_pcd.rviz" />
</launch>
